aries-site/src/examples/components/namevaluelist/data.js

```diff
@@ -0,0 +1,99 @@
+import { Anchor, Box } from 'grommet';
+import { StatusGoodSmall } from 'grommet-icons';
+
+export const alignmentData = {
+  'Created by': (
+    <Anchor label="jane.doe@email.com" href="mailto:jane.doe@email.com" />
```

Comment:
Think we should take the approach of having the data structure as:
```
'Created by': 'jane.doe@email.com'
```
Then, in the render of the example, we can say:
```
if (name === 'Created by') return <Anchor label={value} href={`mailto:${value}`} />
```